py-polars/polars/internals/construction.py

```diff
@@ -92,6 +94,9 @@ def numpy_to_pyseries(
     """
     Construct a PySeries from a numpy array.
     """
+    if not _NUMPY_AVAILABLE:
```

Comment:
Maybe I can leave out these checks and leave this function the same as it was, since a `numpy` array is required to run this function (which is clearly indicated by the function name, too). What do you think?

Reply:
Agree. That check is probably already done higher up the call chain.

Reply:
Added a commit for this 👍